source/core/source/merchants.js

```diff
@@ -229,6 +233,43 @@ var gca_merchants = {
 		}
 	},
 	
+	// Fade items that cost rubies
+	fadeItemsForRubies : {
+		inject : function() {
+			// Save instance
+			var that = this;
+			// Recheck items on item move
+			gca_tools.event.request.onAjaxResponse(function(){
+				that.check();
+			});
+			// Run for the first time
+			this.check();
+		},
+		// Do a check
+		check : function(){
+			
+			// Get shop items
+			var items = document.getElementById('shop').getElementsByClassName("ui-draggable");
+			// For each
+			var t, r;
+			for (var i = items.length - 1; i >= 0; i--) {
+				// Parse tooltip
+				t = items[i].dataset.tooltip.replace(/\./g, "").replace(/\\/g, "");
+				
+				// Get item's rubies
+				r = t.match(/(\d+) <div class="icon_rubies">/);
```

Comment:
Sometimes it is actually easier to just go in and change it! However, I thought you may want me to review it so that you can see what to improve. I will merge it and then make a couple of changes for efficiency (y)

Thanks for looking into this!

Reply:
@GreatApo thank you for the help :).